README.md

```diff
@@ -158,6 +174,31 @@ These are the VS Code [settings] available for the Extension:
 [workspace level]:
   https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings
 
+## Known limitations
+
+Some DVC features have limited support at the moment in the VS Code extension
+(but you can still use them from an IDE's _Terminal_).
+
+### ML Pipelines
+
+The extension examines [`dvc.yaml` files] to identify tracked data and changes,
+but does not currently provide a graphic interface to write or modify stages.
+
+[`dvc.yaml` files]: https://dvc.org/doc/user-guide/project-structure
+
+### Monorepos
+
+While DVC supports Git monorepos with DVC projects inside [sub-directories], the
+extension may have unexpected behaviors when opening such repos. Please open
+each subdir DVC project directly with VS Code instead.
+
+[sub-directories]:
+  https://dvc.org/doc/command-reference/init#initializing-dvc-in-subdirectories
+
+## Roadmap
+
+<!-- ... -->
```

Comment:
Not sure where to find this (⏳) and it overlaps with **Known limitations** somewhat. Should I try to combine them?

Reply:
OK will base this off https://iterative-squad.monday.com/boards/2546691996